geode-redis/src/main/java/org/apache/geode/redis/internal/netty/Coder.java

```diff
@@ -278,6 +281,16 @@ public static ByteBuf getErrorResponse(ByteBufAllocator alloc, String error) {
     return response;
   }
 
+  public static ByteBuf getOOMResponse(ByteBufAllocator alloc, String error) {
+    byte[] errorAr = stringToBytes(error);
+    ByteBuf response = alloc.buffer(errorAr.length + 25);
```

Comment:
The reason that a value of 25 specifically is added to the buffer size here is not clear to me. Other methods in this class, (`getCustomErrorResponse()` for instance) also add 25 to the buffer size, but write fewer bytes to the buffer in total. Is there some way of programmatically determining what the ideal buffer size should be given the expected contents? Failing that, just adding an explanatory comment would go a long way to making the code easier to maintain.

Reply:
This is inherited from the original author(s) and we don't know what this is for. We've also found excessive (and unnecessary) allocations in this class which will shortly be corrected. That change will also remove these weird constants.